wordpress.org/public_html/wp-content/plugins/plugin-directory/email/class-plugin-approved.php

```diff
@@ -29,43 +29,48 @@ function body() {
 To answer some common questions:
 
 * You must use SVN to upload your code -- we are unable to do that for you
-* Your SVN username is %2$s and your password is the same as you use to log in to WordPress.org
-* Your username is case sensitive
+* Your SVN username is `%2$s` and is case sensitive.
+* Generate your SVN password in your [WordPress.org profile](%4$s).
```

Comment:
I'd suggest to make the change more noticiable as this would be the first place some plugin authors will see this, and they need to realize that it now works in a different way.

Maybe something like this:

```
* You can <strong>set up your SVN credentials</strong> (if you haven't already) in the “Account & Security” section of your [WordPress.org profile](%4$s).
* <i>Note: Previously, the credentials to access SVN were your WordPress login and password, this has changed. </i>
```

Reply:
Good idea. I'm thinking overall this email needs an larger update that strongly suggests users use all _new/best-practice_ features (release from tags, svn passwords, release confirmations), I'll open a meta ticket for that.

Reply:
Personally I would avoid language like `Previously...` and focus on new users.

Either they're an existing committer, and they should've been informed / learnt about the change, OR they're a new author who doesn't need to know about the historical aspects of it.

In other words; we should avoid having too much knowledge debt in automated emails (and predefs where possible)

Reply:
I can agree with that, I just think there will be people who need more information to change what they're used to, but I also agree that we don't have to focus on them. 
In that case, just with the first phrase that's fine.
`* You can <strong>set up your SVN credentials</strong> (if you haven't already) in the “Account & Security” section of your [WordPress.org profile](%4$s).
`